llvm/include/llvm/Analysis/LazyValueInfo.h

```diff
@@ -115,6 +115,9 @@ class LazyValueInfo {
   /// PredBB to OldSucc to be from PredBB to NewSucc instead.
   void threadEdge(BasicBlock *PredBB, BasicBlock *OldSucc, BasicBlock *NewSucc);
 
+  /// Inform the analysis cache that we have erased a value.
```

Comment:
For this usage we haven't actually erased a value, so this comment isn't quite right. Maybe "Remove information related to this value from the cache."

Could also call this API forgetValue() rather than eraseValue() to make it more generic.

Reply:
I created `LazyValueInfoCache::forgetValue`.